src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  movies: Movie[],
+  setMovies: (movie: Movie) => void,
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ movies, setMovies }) => {
+  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
+  const [title, setTitle] = useState('');
+
+  const [movieLoadError, setMovieLoadError] = useState(false);
+  const [movieExistError, setMovieExistError] = useState(false);
+
+  const loadMovie = async () => {
+    try {
+      const load = await getMovie(title);
+
+      if (load.Title) {
+        setSelectedMovie(load);
+      }
+    } catch {
+      setSelectedMovie(null);
+      setMovieLoadError(true);
+    }
+  };
+
+  const addMovie = () => {
+    if (selectedMovie) {
+      setSelectedMovie(null);
+      const isUnique = !(movies.find(uniqueMovie => uniqueMovie.imdbID
+        === selectedMovie.imdbID));
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+      if (isUnique) {
+        setMovies(selectedMovie);
+      } else {
+        setMovieExistError(true);
+      }
+    }
+  };
+
```

Comment:
below functionality you should add to your project
If film is not found show an error message below the input. Hide the error after changing a title.